Doc/library/argparse.rst

```diff
@@ -201,9 +201,9 @@ ArgumentParser objects
    * usage_ - The string describing the program usage (default: generated from
      arguments added to parser)
 
-   * description_ - Text to display before the argument help (default: none)
+   * description_ - Text to display before the argument help (default: ``None``)
 
-   * epilog_ - Text to display after the argument help (default: none)
+   * epilog_ - Text to display after the argument help (default: ``None``)
```

Comment:
Here, where the current docs say "default: none", I'm not sure the intent is actually to say "the default is the `None` constant". I think the intent is for it to read more like normal English prose, i.e.: "The `description` parameter determines the text to display before the argument help. The default is for no text to be displayed before the argument help."

Reply:
If this is indeed the case, it could be made more clear and avoid confusion by saying something like `(default is no text)`

Reply:
> If this is indeed the case, it could be made more clear and avoid confusion by saying something like `(default is no text)`

Indeed, the current wording is slightly ambiguous

Reply:
Yes, thanks. That was the change I made on my local branch. I have still been unable to figure out why I can't push. I also don't know how to coax git into displaying some visual that will show me where my pulling, pushing, fetching, and syncing went wrong.

This seems low priority. Give me another day or two of head scratching before taking over. Thanks...